server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMailboxesCreationProcessor.java

```diff
@@ -152,6 +153,16 @@ private void ensureValidMailboxName(MailboxCreateRequest mailboxRequest, Mailbox
         if (name.contains(String.valueOf(pathDelimiter))) {
             throw new MailboxNameException(String.format("The mailbox '%s' contains an illegal character: '%c'", name, pathDelimiter));
         }
+
+        if (nameMatchesSystemMailbox(name)) {
```

Comment:
I don't know the exact specification on that method, but we may think about being idempotent.